addons/udes_stock/models/stock_move.py

```diff
@@ -194,6 +205,9 @@ def split_out_move_lines(self, move_lines):
             if self.move_orig_ids:
                 (bk_move | self).update_orig_ids(self.move_orig_ids)
 
+            incomplete_moves._recompute_state()
+            move_lines.write({"state": bk_move.state})
```

Comment:
Is this needed?

Reply:
Looks like not any more- left it from the rebase